src/enumerable.cr

```diff
@@ -884,6 +884,56 @@ module Enumerable(T)
     reduce(initial) { |memo, e| memo + (yield e) }
   end
 
+  # Multiplies all the elements in the collection together.
+  #
+  # Only collections of numbers (objects that can be multiplied via a `*` method) are supported.
+  #
+  #     [1, 2, 3, 4, 5, 6].multiply  #=> 720
```

Comment:
Good catch, yes.  I've fixed it locally.  Let me know if I should push that change.